dask/dataframe/io/parquet/arrow.py

```diff
@@ -908,8 +908,16 @@ def initialize_write(
 
         return fmd, i_offset
 
-    @staticmethod
+    @classmethod
+    def _pandas_to_arrow_table(
+        cls, df: pd.DataFrame, preserve_index=False, schema=None
+    ) -> pa.Table:
+        table = pa.Table.from_pandas(df, preserve_index=preserve_index, schema=schema)
+        return table
+
+    @classmethod
```

Comment:
I think this can still be a staticmethod right?

Reply:
I think it needs to be a classmethod, because I use `cls._pandas_to_arrow_table`

(also, in the FastParquetEngine, this was already changed in https://github.com/dask/dask/pull/6211, so it might have been an oversight it was not yet changed here

Reply:
Ah ok that makes sense. Plus if `FastParquetEngine` already does it then I think it's fine.